code/network/multi_ingame.cpp

```diff
@@ -475,7 +475,7 @@ static int Multi_ingame_timer_coords[GR_NUM_RESOLUTIONS][2] = {
 //#define MULTI_INGAME_TIME_LEFT_Y			411
 
 #define MULTI_INGAME_TIME_SECONDS		(1000 * 15)
```

Comment:
I recommend setting this to just 15 and then using `MILLISECONDS_PER_SECOND` when the actual timestamp is created.